### PR TITLE
#15 background

### PR DESCRIPTION
**Pull request by felipeloha**
#15 background

### CODE REVIEW

File: src/components/Main.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
 import React from 'react';
-import img1 from '../img/Manos_grande.png';
+import img1 from '../img/Illegal.png';
```
Comment (ZsuzsaMano) [inline src/components/Main.js:2]: It was requested from the group to put the image with the hands. I think we should ask before changing it.

Comment (JacqueGM) [reply src/components/Main.js:2]: ahh I didn't know that, maybe yeah it's a good idea ask them but maybe we can change it and put it in materials for now and merge still?

Comment (felipeloha) [reply src/components/Main.js:2]: we can send them the link and they can decide.

Comment (ZsuzsaMano) [reply src/components/Main.js:2]: Can you please do that? Maybe just a screenshot from both? Thx

**Pull request merged**